tensorflow/core/kernels/sample_distorted_bounding_box_op.cc

```diff
@@ -275,6 +277,22 @@ class SampleDistortedBoundingBoxOp : public OpKernel {
                     "bounding boxes must have shape [4] or [*, 4], got ",
                     input_boxes.shape().DebugString()));
 
+    if (context->num_inputs() == 3) {
+      const Tensor& min_object_covered = context->input(2);
+
+      OP_REQUIRES(
+          context, TensorShapeUtils::IsScalar(min_object_covered.shape()),
+          errors::InvalidArgument("min_object_covered must be 0-D, got shape ",
+                                  min_object_covered.shape().DebugString()));
+
+      min_object_covered_ = min_object_covered.scalar<float>()();
```

Comment:
Compute() can be run by multiple threads, so this is going to result in a potential memory race.  I would suggest doing something like:

float min_object_recovered_val = 0.0;
if (context->num_inputs() == 3) {
  ...
  min_object_recovered_val = min_object_recovered.scalar<float>()();
} else {
  min_object_recovered_val = min_object_recovered_;
}

and then use that float below.